src/js/ripple/orderbook.js

```diff
@@ -439,19 +439,22 @@ OrderBook.prototype.setFundedAmount = function(offer, fundedAmount) {
     return offer;
   }
 
-  var isOfferGetsExceeded = Amount.from_json(fundedAmount)
+  var isOfferGetsExceeded = Amount.from_json(
+    this._currencyGets.is_native()
+    ? fundedAmount
+    : fundedAmount + OrderBook.IOU_SUFFIX
```

Comment:
Can you please let me know the case that won't work? I'm going to create a JIRA task for this and I want to document any risks.